tasks/main.yml

```diff
@@ -34,12 +34,32 @@
     - name: Gather OpenStack infrastructure information
       command: openstack stack output show {{ cluster_name }} cluster_group -f json
       register: stack_output
+      changed_when: false
 
     - name: Extract node groups
       set_fact:
         cluster_group: "{{ stack_output.stdout | from_json }}"
   when: cluster_state == 'query'
 
+- name: Extract server ids from cluster group output
+  set_fact:
+    cluster_servers: "{{ cluster_group.output_value | json_query('[*].nodes[*].id') | flatten }}"
```

Comment:
json_query requires jmespath to be installed. Could you add that to the readme?

Reply:
Done